src/pages/ProfilesExplorerView/components/SceneAiPanel/components/AiButton/AIButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { css } from '@emotion/css';
```

Comment:
This component, as well as other files have been copied from the "shared" folder and customized for the Scenes app. Once we remove the code for the legacy Comparison pages (which is still present), we'll clean up everything.